ci-operator/jobs/openshift/cluster-api-provider-openstack/openshift-cluster-api-provider-openstack-master-presubmits.yaml

```diff
@@ -74,6 +80,20 @@ presubmits:
         projected:
           sources:
           - secret:
+              items:
+              - key: clouds.yaml
+                path: clouds.yaml
+              name: cluster-secrets-openstack-vexxhost
+          - secret:
+              items:
+              - key: .awscred
+                path: .awscred
```

Comment:
Are these supposed to be there?

Reply:
Yes, we use them to setup DNS with route53